youtube_dl/extractor/crunchyroll.py

```diff
@@ -430,7 +430,10 @@ def _real_extract(self, url):
         if note_m:
             raise ExtractorError(note_m)
 
-        mobj = re.search(r'Page\.messaging_box_controller\.addItems\(\[(?P<msg>{.+?})\]\)', webpage)
+        mobj = re.search(
+            r'Page\.messaging_box_controller\.addItems\(\[(?P<msg>{(?!.*},{).+?})\]\)',
```

Comment:
Valid JS could include white space around `.`, `(`, `)`, `[`, `]`, and `,`, though it makes the RE a bit messier, something like this:
```
            r'Page\s*\.\s*messaging_box_controller\s*\.\s*addItems\s*\(\s*\[\s*(?P<msg>{(?!.*}\s*,\s*{).+?})\s*\]\s*\)'
```